README.rst

```diff
@@ -44,10 +44,23 @@ Installation
 ------------
 
 .. begin-installation-marker
+0. If you have an existing installation of an old COBRA toolbox on your system, please remove 
```

Comment:
May you start this counter with `1.`?

Reply:
It was to some extent intentional to start with 0, as this is only required if there is an old toolbox. 
Starting with sounds as if it is a step always required.

Reply:
Then this should be a different number, or explicitly stating that this is an optional step.